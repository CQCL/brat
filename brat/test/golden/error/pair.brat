row :: Int, Bool
row = 4, true

pair :: [Int, Bool]
pair = row -- Distinct from a pair
