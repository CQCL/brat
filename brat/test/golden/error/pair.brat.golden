Error in pair@FC {start = Pos {line = 5, col = 8}, end = Pos {line = 5, col = 11}}:
  Type mismatch when checking row
Expected: (a1 :: Pair(Int, Bool))
But got:  (a1 :: Int), (b1 :: Bool)


